### ISSUE TITLE
Payments field blank

### ISSUE THREAD

**Issue by c-hri-s** [philipmulcahy/azad#62]
Hi, 

Previously the payments field would indicate which credit card (or other method) was used for payment. This is really useful for me to reconcile.

Using 1.8.24 on amazon.co.uk the payment column is blank except for a couple of items that say 'pending'.

**Comment by philipmulcahy** [comment]
1.8.26 seems to have fixed this

**Comment by philipmulcahy** [comment]
pending is probably my/our fault
missing CC info is a symptom of a bit of cat and mouse between Amazon and the extension - they are making it increasingly hard to get this info.

**Comment by c-hri-s** [comment]
This has broken again with the latest release/Amazon changes.
From a list of 181 transactions in 2019 I only have payment details for one item, all the rest are blank

**Comment by philipmulcahy** [comment]
Gah! I will look this weekend

**Comment by c-hri-s** [comment]
Sorry - ignore that (I deleted the comment, but obviously you got notification).

I started the scan, but then quote quickly changed the display from '10' to 'All' items. Doing so seemed to stop the background scan, so although I saw the data none of the payment details were populated. Rerunning it gave the same results so I made the post.

Clearing the cache and rerunning the scan - and leaving it on the '10' items page, let the scan complete and the details were there.

Sorry for the false alarm.
